### ISSUE TITLE
bug: java.nio.file.InvalidPathException: Illegal char <<> at index 0: <719a309c-7786-4dcc-ad08-2a3da57e7d58>/modules/msthemeconfig...

### ISSUE THREAD

**Issue by ijzershop** [sourcegraph/jetbrains#2594]
### IDE Information

PhpStorm 2024.2.4
Build #PS-242.23726.107, built on October 23, 2024
Licensed to moderne smid / jelmer stoker
You have a perpetual fallback license for this version.
Subscription is active until November 5, 2025.
Runtime version: 21.0.4+13-b509.26 amd64 (JCEF 122.1.9)
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Toolkit: sun.awt.windows.WToolkit
Windows 11.0
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 4022M
Cores: 16
Registry:
  debugger.new.tool.window.layout=true
  run.processes.with.pty=TRUE
  ide.experimental.ui=true
  i18n.locale=
Non-Bundled Plugins:
  de.espend.idea.localization (0.0.6)
  com.berzanmikaili.intellij.monokai_pro (2.0.2c)
  com.intellij.ideolog (242.23726.110)
  com.github.timo_reymann.composer_dump_autoload_phpstorm_plugin (2.1.1)
  ru.adelf.idea.dotenv (2024.2.1)
  de.espend.idea.php.toolbox (6.2.0)
  de.espend.idea.php.annotation (11.1.0)
  com.github.dinbtechit.vscodetheme (1.10.12)
  com.sourcegraph.jetbrains (7.0.18)
  mobi.hsz.idea.gitignore (4.5.4)

### Bug Description

? 

### Additional context

Stacktrace:
```text
java.nio.file.InvalidPathException: Illegal char <<> at index 0: <719a309c-7786-4dcc-ad08-2a3da57e7d58>/modules/msthemeconfig/controllers/front/ajax.php
	at java.base/sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:199)
	at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:175)
	at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
	at java.base/sun.nio.fs.WindowsPath.parse(WindowsPath.java:92)
	at java.base/sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:231)
	at com.sourcegraph.cody.agent.protocol.ProtocolTextDocument$Companion.uriFor(ProtocolTextDocument.kt:197)
	at com.sourcegraph.cody.agent.protocol.ProtocolTextDocument$Companion.fromEditorWithOffsetSelection(ProtocolTextDocument.kt:92)
	at com.sourcegraph.cody.listeners.CodyCaretListener.caretPositionChanged(CodyCaretListener.kt:27)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at com.intellij.util.EventDispatcher.dispatchVoidMethod(EventDispatcher.java:119)
	at com.intellij.util.EventDispatcher.lambda$createMulticaster$1(EventDispatcher.java:84)
	at jdk.proxy2/jdk.proxy2.$Proxy124.caretPositionChanged(Unknown Source)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at com.intellij.util.EventDispatcher.dispatchVoidMethod(EventDispatcher.java:119)
	at com.intellij.util.EventDispatcher.lambda$createMulticaster$1(EventDispatcher.java:84)
	at jdk.proxy2/jdk.proxy2.$Proxy124.caretPositionChanged(Unknown Source)
	at com.intellij.openapi.editor.impl.CaretModelImpl.fireCaretPositionChanged(CaretModelImpl.java:536)
	at com.intellij.openapi.editor.impl.CaretImpl.doMoveToVisualPosition(CaretImpl.java:593)
	at com.intellij.openapi.editor.impl.CaretImpl.lambda$moveToVisualPosition$4(CaretImpl.java:531)
	at com.intellij.openapi.editor.impl.CaretModelImpl.doWithCaretMerging(CaretModelImpl.java:412)
	at com.intellij.openapi.editor.impl.CaretImpl.moveToVisualPosition(CaretImpl.java:531)
	at com.intellij.openapi.editor.impl.CaretImpl.moveToVisualPosition(CaretImpl.java:527)
	at com.intellij.openapi.editor.CaretModel.moveToVisualPosition(CaretModel.java:68)
	at com.intellij.openapi.editor.impl.EditorImpl$MyMouseAdapter.processMousePressed(EditorImpl.java:4498)
	at com.intellij.openapi.editor.impl.EditorImpl$MyMouseAdapter.lambda$runMousePressedCommand$0(EditorImpl.java:4288)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:226)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:178)
	at com.intellij.openapi.editor.impl.EditorImpl$MyMouseAdapter.runMousePressedCommand(EditorImpl.java:4293)
	at com.intellij.openapi.editor.impl.EditorImpl$MyMouseAdapter.mousePressed(EditorImpl.java:4207)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6662)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3394)
	at java.desktop/java.awt.Component.processEvent(Component.java:6430)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5035)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4860)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4963)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4574)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4518)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2810)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4860)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:783)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:728)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:98)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:755)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:753)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:752)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:696)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.kt:635)
	at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$14(IdeEventQueue.kt:581)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:84)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:581)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:73)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1$1.compute(IdeEventQueue.kt:357)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1$1.compute(IdeEventQueue.kt:356)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrior...

**Comment by PriNova** [comment]
Duplicate of #2199
